Mathlib/Algebra/Order/Group/Abs.lean

```diff
@@ -254,7 +254,8 @@ variable [Group α] [LinearOrder α] {a b : α}
 @[to_additive] lemma mabs_eq_mabs : |a|ₘ = |b|ₘ ↔ a = b ∨ a = b⁻¹ := by
   refine' ⟨fun h ↦ ?_, by rintro (h | h) <;> simp [h, abs_neg]⟩
   obtain rfl | rfl := eq_or_eq_inv_of_mabs_eq h <;>
-    simpa only [inv_eq_iff_eq_inv (a := |b|ₘ), inv_inj, or_comm] using mabs_choice b
+    simpa only [inv_eq_iff_eq_inv (a := |b|ₘ), inv_inv, inv_inj, or_comm] using mabs_choice b
+  -- v4.7.0-rc1 issues
```

Comment:
@mattrobball what is the intent of this comment? It looks fine? Did you want to investigate further?

Reply:
I annotated anything slightly off to make sure it would get a second look. Feel free to remove any if you are satisfied with the diff.